it/google-cloud-platform/src/main/java/com/google/cloud/teleport/it/gcp/LoadTestBase.java

```diff
@@ -403,6 +413,27 @@ protected Map<String, Double> getSystemLatencyMetrics(LaunchInfo launchInfo)
     return systemLatencyMetrics;
   }
 
+  /** Gets the time interval when workers were active to be used by monitoring client. */
+  // TODO (pranavbhandari): This does not work with Runner V2, due to different worker messages.
```

Comment:
I checked both runner v1 and v2 has same message.
"All workers have finished the startup processes and began to receive work requests"
and
"Stopping worker pool"

if it's not working could it be some issue elsewhere?

Reply:
You are right! I tested this change on a runner v2 job and it works.